es-vue-base/src/lib-components/EsFooter.vue

```diff
@@ -0,0 +1,156 @@
+                    </div>
+                    <!-- Social icons -->
+                    <div class="d-flex text-gray-700">
+                        <a
+                            v-for="iconLink in socialLinks"
+                            :key="iconLink"
+                            class="text-reset pr-150"
+                            :href="iconLink.url">
+                            <span class="sr-only">{{ iconLink.text }}</span>
+                            <component :is="iconLink.icon" />
+                        </a>
+                    </div>  <!-- Social icons -->
+                </div>  <!-- Logo and Social -->
+            </div>
+            <hr class="border-top border-dark m-0">
+            <!-- Trademark Info -->
+            <p class="mt-150 mt-lg-200">
+                ENERGYSAGE is a registered trademark and the EnergySage logo is a trademark of EnergySage, Inc.
+                Other trademarks are the property of either EnergySage, Inc. or our licensors
+                and are used with permission.
+            </p>
+            <p class="mb-200">
+                © Copyright 2009-{{ new Date().getFullYear() }} EnergySage, Inc. All rights reserved.
+            </p>
+            <!-- Trademark Info -->
+            <!-- Legal -->
+            <div class="row justify-content-between mb-150">
+                <div
+                    v-for="link in legalLinks"
+                    :key="link"
+                    class="col col-6 col-lg font-size-50 mt-25 mb-50">
+                    <a
+                        :href="link.url"
+                        class="text-reset font-weight-normal">
+                        {{ link.text }}
+                    </a>
+                </div>
+            </div>  <!-- Legal -->
+            <!-- DOE -->
+            <div class="d-lg-flex align-items-end">
+                <!-- eslint-disable-next-line -->
+                <img
+                    src="https://www-static.energysage.com/static/img/doe/doe-logo-179.943fe6467b04.png"
+                    aria-hidden="true"
+                    class="mr-100 mb-100 mb-lg-0"
+                    width="99"
+                    height="25">
+                <p class="mb-0">
+                    <a href="https://www.energy.gov/eere/solar/articles/eere-success-story-doe-funding-helps-build-one-stop-shop-rooftop-pv-systems">
+                        Learn more about our success working with the US. Department of Energy.
+                    </a>
+                </p>
+            </div>  <!-- DOE -->
+        </div>
+    </div>
+</template>
+
+<script lang="js">
+export default {
+    name: 'EsFooter',
+    data() {
+        return {
+            columns: {
```

Comment:
Optional: In cases where we care about preserving the order of a set of items in JavaScript, I'd generally still recommend using an array, even though in most modern browser implementations, the order of chronological definition will actually be preserved and this will work. So not a necessary change but I did change the nav data objects to be arrays just for extra precaution.
https://www.stefanjudis.com/today-i-learned/property-order-is-predictable-in-javascript-objects-since-es2015/